src/browser/api/application.js

```diff
@@ -348,7 +348,7 @@ Application.grantWindowAccess = function() {
 };
 Application.isRunning = function(identity) {
     let uuid = identity && identity.uuid;
-    return uuid && coreState.getAppRunningState(uuid) && !coreState.getAppRestartingState(uuid);
+    return !!(uuid && coreState.getAppRunningState(uuid) && !coreState.getAppRestartingState(uuid));
```

Comment:
Looks like this got carried over on the rebase. Can you remove this

Reply:
my apologies, this is reverted.